lib/danger/danger_core/plugins/dangerfile_gitlab_plugin.rb

```diff
@@ -0,0 +1,73 @@
+require 'danger/plugin_support/plugin'
+
+module Danger
+  class DangerfileGitLabPlugin < Plugin
+    def self.new(dangerfile)
+      return nil if dangerfile.env.request_source.class != Danger::RequestSources::GitLab
+
+      super
+    end
+
+    def initialize(dangerfile)
+      super(dangerfile)
+
+      @gitlab = dangerfile.env.request_source
+    end
+
+    # @!group PR Metadata
+    # The title of the Pull Request
+    # @return String
+    #
+    def pr_title
```

Comment:
IMO, I don't think so, for `Danger::RequestSources::GitLab` there should be, ( you could add method aliases though, so that people can C&P examples that may reference the github names? I'd be on board with that. )